services/mirror/mirror.go

```diff
@@ -100,7 +100,9 @@ func SaveAddress(m *models.Mirror, addr string) error {
 	}
 
 	_, err = git.NewCommand("remote", "add", "origin", "--mirror=fetch", addr).RunInDir(repoPath)
```

Comment:
lint fails on this line. `err` is never checked.

Reply:
noticed, fixed ...